apps/web/src/env.ts

```diff
@@ -4,14 +4,21 @@ import { createEnv } from '@t3-oss/env-nextjs'
 import { z } from 'zod'
 
 export default createEnv({
+  skipValidation: process.env.BUILDING_CONTAINER == 'true',
```

Comment:
so this did the trick

Reply:
I think so. This should be in the next PR but I'll keep in this one